### PR TITLE
(Core/Commands): Introduce .npc guid

### PR DESCRIPTION
**Pull request by neifion-00000000**
(Core/Commands): Introduce .npc guid
.npc info is too robust and floods the screen
.npc guid will simply provide only the first part of .npc info's output

## Changes Proposed:
-  Creates a new command named .npc guid which outputs only the first portion of .npc info

## Issues Addressed:

- using .npc info to get simple information about creatures was too difficult to read in game, this begins to address that, though some may find it still outputs too much info

## How to Test the Changes:

1. After compile, enter world on any char
2. use .npc guid on a creature
3. use .npc guid on yourself to make sure error outputs are correct

## Known Issues and TODO List:

- despite its name, it outputs a decent amount more than guid, I figure it can either be renamed or shortened more later and that I should keep my PR scope small especially since this is my first c++ commit on github

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Nyeriah** [comment]
Missing the sql part (command table), otherwise looks good

**Comment by neifion-00000000** [comment]
not sure what the codestyle issues are

File: src/server/scripts/Commands/cs_npc.cpp [hunk 1]
```diff
@@ -574,7 +575,7 @@ class npc_commandscript : public CommandScript
         handler->PSendSysMessage(LANG_CREATURE_FOLLOW_YOU_NOW, creature->GetName().c_str());
         return true;
     }
-
+
```
Comment (Nyeriah) [inline src/server/scripts/Commands/cs_npc.cpp:578]: whitespaces here

**Comment by Nyeriah** [comment]
> not sure what the codestyle issues are (tho I'm sure they're my fault)

Left a comment

File: src/server/scripts/Commands/cs_npc.cpp [hunk 2]
```diff
@@ -651,6 +652,37 @@ class npc_commandscript : public CommandScript
...
+        Creature* target = handler->getSelectedCreature();
+
+        if (!target)
+        {
+            handler->SendSysMessage(LANG_SELECT_CREATURE);
+            handler->SetSentErrorMessage(true);
+            return false;
+        }
+
+        CreatureTemplate const* cInfo = target->GetCreatureTemplate();
```
Comment (Nyeriah) [inline src/server/scripts/Commands/cs_npc.cpp:666]: this variable is never being used for anything

**Comment by neifion-00000000** [comment]
awesome thanks for the help, fingers crossed everything else looks like it gets used

File: data/sql/updates/pending_db_world/rev_1658266226251855400.sql [hunk 3]
```diff
@@ -0,0 +1,3 @@
+--
+DELETE FROM `command` WHERE `name` = 'npc guid';
+INSERT INTO `command` (`name`, `security`, `help`) VALUES ('npc guid', 1, 'Synatx: .npc guid\r\n\r\nDisplays GUID, faction, NPC flags, Entry ID, Model ID for selected creature.');
```
Comment (Nyeriah) [inline data/sql/updates/pending_db_world/rev_1658266226251855400.sql:3]:
```suggestion
INSERT INTO `command` (`name`, `security`, `help`) VALUES ('npc guid', 1, 'Syntax: .npc guid\r\n\r\nDisplays GUID, faction, NPC flags, Entry ID, Model ID for selected creature.');
```

**Review by Nyeriah — APPROVED** [review]

**Pull request merged**